Changelog.md

```diff
@@ -0,0 +1,7 @@
+## [Unreleased] (07/01/23)
+
+### Added
+- Read and sort json files
```

Comment:
How do we sort them? Not against having this point though.

Reply:
done